env-vars.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const dotenv = require('dotenv');
+
+module.exports = async () => {
+  return Object.entries(dotenv.config({ path: '.env' }).parsed).reduce((acc, [key, value]) => {
+    if (!key.startsWith('ETHERSCAN_API_KEY')) {
+      acc[key] = value;
```

Comment:
I would avoid mutating the acc object and instead to: `return { ...acc, [key]: value };`